fv-tests/server_test.go

```diff
@@ -287,6 +287,11 @@ var _ = Describe("With an in-process Server", func() {
 			expectedEndState[path] = update
 			decoupler.OnUpdates([]api.Update{update})
 		}
+		return expectedEndState
+	}
+
+	sendNUpdatesThenInSync := func(n int) map[string]api.Update {
+		expectedEndState := sendNUpdates(n)
```

Comment:
Why this change?

Reply:
I made a test locally that used this change but I didn't think it was worth pushing in its current form.  OTOH, this small refactoring would allow that kind of test to be written more easily in future so I kept it.